tests/phpunit/api/v3/CountryTest.php

```diff
@@ -104,4 +104,34 @@ public function testCreateDuplicateFail() {
     $this->assertEquals(1, $check);
   }
 
+  /**
+   * Test that the list of states is in the correct format when chaining
+   * and using sequential.
+   */
+  public function testCountryStateChainSequential() {
+    $result = $this->callAPISuccess('Country', 'getsingle', [
+      'iso_code' => 'US',
+      'api.Address.getoptions' => [
+        'field' => 'state_province_id',
+        'country_id' => '$value.id',
+        'sequential' => 1,
+      ],
+    ]);
+    $this->assertSame(['key' => 1000, 'value' => 'Alabama'], $result['api.Address.getoptions']['values'][0]);
+    $this->assertSame(['key' => 1001, 'value' => 'Alaska'], $result['api.Address.getoptions']['values'][1]);
+    $this->assertSame(['key' => 1049, 'value' => 'Wyoming'], $result['api.Address.getoptions']['values'][59]);
```

Comment:
why are there 59 Keys here?

Reply:
Because there are some territories like American Samoa that aren't technically states.

Reply:
But they shoud be  - the revolution starts here. Equal rights for all parts